kpops/component_handlers/kafka_connect/model.py

```diff
@@ -123,3 +125,10 @@ class KafkaConnectorResetterConfig(CamelCaseConfigModel):
 class KafkaConnectorResetterValues(HelmAppValues):
     connector_type: Literal["source", "sink"]
     config: KafkaConnectorResetterConfig
+    image_tag: str = Field(default="latest")
```

Comment:
inherit from `StreamsBootstrapValues` instead. Then you can drop the field and validator here

Reply:
I didn't do this intentionally, because I don't think that `KafkaConnectorResetterValues` is or belongs to `StreamsBootstrapValues` and should be its own value.

Reply:
ah, I wasn't aware. It's just a little cumbersome to have the code duplication for it. do you think we could extract it to a shared base easily?

Reply:
but perhaps also we won't need the resetter much longer with the recent API changes we already talked about #379

Reply:
Yeah I think so too. We can have it removed and keep it for streams app only